docs/overview/resources.mdx

```diff
@@ -24,11 +24,7 @@ We curate this list to only contain resources that are still up to date.
 ### Apps
 
 - [mirumee: `saleor-app-framework-python`](https://github.com/mirumee/saleor-app-framework-python)
-- [saleor: `saleor-app-search`](https://github.com/saleor/saleor-app-search)
-- [saleor: `saleor-app-mailing`](https://github.com/saleor/saleor-app-mailing)
-- [saleor: `saleor-app-klaviyo`](https://github.com/saleor/saleor-app-klaviyo)
-- [saleor: `saleor-app-slack`](https://github.com/saleor/saleor-app-slack)
-- [saleor: `saleor-app-cms-hub`](https://github.com/saleor/saleor-app-cms-hub)
+- [saleor: `saleor/apps`](https://github.com/saleor/apps)
```

Comment:
I like the change, but it makes vendor brands harder to find. Maybe we could add a list of supported integrations somewhere instead of having a generic "resources" page?

Reply:
The list of available apps is present here https://docs.saleor.io/docs/3.x/developer/app-store/apps/overview and will be improved